src/Handlers/MySQLHandler.php

```diff
@@ -0,0 +1,35 @@
+<?php 
+
+namespace Handlers;
+
+class MySQLHandler {
+
+    private $connection;
+
+    private static function connectDatabase()
+    {
+        // database config
+        $host="localhost";
+        $user="root";
+        $pass="";
+        $db="shortered";     
+        
+        $connection = new mysqli(
+             $host,
+             $user,
+             $pass,
+             $db
+        );
+        
+        if (mysqli_connect_error()) {
+             die("Connection failed: " . mysqli_connect_error());
+        }
+        echo "Connected succesfully.";
+
+        return $connection;
+    }
+
+    public function __costruct() {
```

Comment:
Would be better if you place the `__contruct` method at the top of the class, I mean it should be first.